### ISSUE TITLE
Import of newrelic_one_dashboard fails with tables + Filter Current dashboard flag

### ISSUE THREAD

**Issue by Nordes** [newrelic/terraform-provider-newrelic#1172]
### Terraform Version
Terraform v0.14.2
+ provider registry.terraform.io/newrelic/newrelic v2.17.0

### Affected Resource(s)
Please list the resources as a list, for example:
- `newrelic_one_dashboard`

### Terraform Configuration
No configuration (Import issue)

![image](https://user-images.githubusercontent.com/446572/106959169-5184f780-6708-11eb-932a-f9994817e2d4.png)

### Actual Behavior
What actually happened? **Error** 

```sh
newrelic/playground $ rm -f terraform.tfstate &&  terraform import newrelic_one_dashboard.this MjkzMzM2NXxWSVp8REFTSEJPQVJEfGRhOjEwNzA5OA
newrelic_one_dashboard.this: Importing from ID "MjkzMzM2NXxWSVp8REFTSEJPQVJEfGRhOjEwNzA5OA"...
newrelic_one_dashboard.this: Import prepared!
  Prepared newrelic_one_dashboard for import
newrelic_one_dashboard.this: Refreshing state... [id=MjkzMzM2NXxWSVp8REFTSEJPQVJEfGRhOjEwNzA5OA]

Error: Invalid address to set: []string{"page", "0", "widget_table", "2", "linked_entity_guids"}
```

### Expected Behavior
I would expect the import to work like when I uncheck the option in the UX. (one.newrelic.com)

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create a dashboard with a NRQL request such as: `SELECT count(*), uniques(appName)  FROM TransactionError FACET transactionName SINCE 24 hours ago UNTIL now` displayed as table.
2. Tick the box `filter the current dashboard` (green check)
3. Go back to your console after noting the dashboard id
4.  `terraform init`
5. `terraform import newrelic_one_dashboard.this <some guid here>`

Basically, it's an empty terraform project with only the newrelic provider + the resource named as mentioned in the import.

workaround to make it work: Simply untick the option and all goes ok, except that it's not really what I expected to do.

### Debug Output
https://gist.github.com/Nordes/82bb8570aaa6407fd61898c56ddf574c

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Important Factoids
N/A

### References
N/A

**Comment by jthurman42** [comment]
The `widget_table` schema does not include `linked_entities`, though it is returned by the API.  I can reproduce this, and started a PR to resolve it and another dashboard issue.
